<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="moshelevin-lt" failures="5" tests="8" name="Test" time="1609.345" errors="0" timestamp="24 Jan 2018 13:56:25 GMT">
  <testcase classname="seetest.IOS_appstoretest" name="runtest" time="92.225">
    <failure type="com.experitest.client.InternalException" message="Exception caught while executing click: Failed to click &amp;apos;Face&amp;apos;. Object repository doesn&amp;apos;t contain: default\Face">
      <![CDATA[com.experitest.client.InternalException: Exception caught while executing click: Failed to click 'Face'. Object repository doesn't contain: default\Face
	at com.experitest.client.JavaClientInternals.executeInternally(JavaClientInternals.java:249)
	at com.experitest.client.Client.execute(Client.java:247)
	at com.experitest.client.Client.click(Client.java:476)
	at seetest.IOS_appstoretest.runtest(IOS_appstoretest.java:87)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.base/java.lang.Thread.run(Thread.java:844)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- runtest -->
  <testcase classname="seetest.Test6_mytest1" name="runtest" time="108.785">
    <failure type="com.experitest.client.InternalException" message="Exception caught while executing clickIn: Element was not identified: search element: &amp;apos;xpath=//*[@id=&amp;apos;text1&amp;apos;]&amp;apos;">
      <![CDATA[com.experitest.client.InternalException: Exception caught while executing clickIn: Element was not identified: search element: 'xpath=//*[@id='text1']'
	at com.experitest.client.JavaClientInternals.executeInternally(JavaClientInternals.java:249)
	at com.experitest.client.Client.execute(Client.java:247)
	at com.experitest.client.Client.clickIn(Client.java:559)
	at seetest.Test6_mytest1.runtest(Test6_mytest1.java:47)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.base/java.lang.Thread.run(Thread.java:844)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- runtest -->
  <testcase classname="seetest.PlaystoreTest" name="runtest" time="109.113">
    <failure type="com.experitest.client.InternalException" message="Exception caught while executing click: Element was not identified: &amp;apos;xpath=//*[@text=&amp;apos;TOP CHARTS&amp;apos;]&amp;apos; at zone NATIVE">
      <![CDATA[com.experitest.client.InternalException: Exception caught while executing click: Element was not identified: 'xpath=//*[@text='TOP CHARTS']' at zone NATIVE
	at com.experitest.client.JavaClientInternals.executeInternally(JavaClientInternals.java:249)
	at com.experitest.client.Client.execute(Client.java:247)
	at com.experitest.client.Client.click(Client.java:476)
	at seetest.PlaystoreTest.runtest(PlaystoreTest.java:45)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.base/java.lang.Thread.run(Thread.java:844)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- runtest -->
  <testcase classname="seetest.TestMyapp" name="runtest" time="129.381"/>
  <testcase classname="seetest.Test7_mytest2" name="runtest" time="147.899"/>
  <testcase classname="seetest.Test5_mytest" name="runtest" time="175.594"/>
  <testcase classname="seetest.EribankLogin" name="runtest" time="213.494">
    <failure type="java.lang.StringIndexOutOfBoundsException" message="begin 18, end 6, length 8">
      <![CDATA[java.lang.StringIndexOutOfBoundsException: begin 18, end 6, length 8
	at java.base/java.lang.String.checkBoundsBeginEnd(String.java:3116)
	at java.base/java.lang.String.substring(String.java:1885)
	at seetest.EribankLogin.runtest(EribankLogin.java:124)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.base/java.lang.Thread.run(Thread.java:844)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- runtest -->
  <testcase classname="seetest.ChromeAndSafari" name="runtest" time="478.964">
    <failure type="com.experitest.client.InternalException" message="Exception caught while executing click: Element was not identified: &amp;apos;xpath=//div[@class=&amp;quot;nav-menu__hamburger&amp;quot;]&amp;apos; at zone WEB">
      <![CDATA[com.experitest.client.InternalException: Exception caught while executing click: Element was not identified: 'xpath=//div[@class="nav-menu__hamburger"]' at zone WEB
	at com.experitest.client.JavaClientInternals.executeInternally(JavaClientInternals.java:249)
	at com.experitest.client.Client.execute(Client.java:247)
	at com.experitest.client.Client.click(Client.java:476)
	at seetest.ChromeAndSafari.runtest(ChromeAndSafari.java:138)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.base/java.lang.Thread.run(Thread.java:844)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- runtest -->
</testsuite> <!-- Test -->
